reporter/src/main/kotlin/Reporter.kt

```diff
@@ -59,9 +60,11 @@ interface Reporter {
      * The report may be post-processed by a [postProcessingScript] before it is written to [outputStream] whereas the
      * [licenseConfiguration] is passed as a parameter to that script.
      */
+    @Suppress("LongParameterList")
     fun generateReport(
         outputStream: OutputStream,
         ortResult: OrtResult,
+        config: OrtConfiguration = OrtConfiguration(),
```

Comment:
I plan to replace that anyway by making the `OrtConfiguration` globally available in a separate PR.

Reply:
Ideally, the addition of this parameter would have been done in a preparing commit, so that this commit really only add the new notice reporter implementation, like the commit message says.

Reply:
Done.